aspnetcore/migration/22-to-30.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Migrate from ASP.NET Core 2.2 to 3.0 Preview
+author: tdykstra
+description: Learn how to migrate an ASP.NET Core 2.2 project to ASP.NET Core 3.0.
+ms.author: tdykstra
+ms.custom: mvc
+ms.date: 01/04/2019
+uid: migration/22-to-30
+---
+# Migrate from ASP.NET Core 2.2 to 3.0 Preview
+
+By [Scott Addie](https://github.com/scottaddie)
+
+This article explains how to update an existing ASP.NET Core 2.2 project to ASP.NET Core 3.0.
+
+[!INCLUDE[](~/includes/net-core-prereqs-all-3.0.md)]
+
+## Update the project file
+
+* Set the [Target Framework Moniker (TFM)](/dotnet/standard/frameworks#referring-to-frameworks) to `netcoreapp3.0`:
+
+  ```xml
+  <TargetFramework>netcoreapp3.0</TargetFramework>
+  ```
+
+* Remove any `<PackageReference>` to [Microsoft.AspNetCore.All](xref:fundamentals/metapackage)` or [Microsoft.AspNetCore.App](xref:fundamentals/metapackage-app).
```

Comment:
AFAIK (without much discussion on it), Rick, Scott, and I adopted a pattern while you were away. The links to the metapackage topics will say "metapackage" in the link, so I suggest (and there's a stray backtick there, too) ...

```suggestion
* Remove any `<PackageReference>` to [Microsoft.AspNetCore.All metapackage](xref:fundamentals/metapackage) or [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app).
```